### PR TITLE
Moving to Hooks

### PR DESCRIPTION
**Pull request by Himadu2000**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Do anything you like and pull request or notify me, the Author Himadu Sirimanna. Please remember to follow community guidelines.
```
Comment (ngoyal16) [inline CONTRIBUTING.md:1]: please put original author name

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,24 @@
-MIT License
+This is free and unencumbered software released into the public domain.
```
Comment (ngoyal16) [inline LICENSE:1]: use MIT License only

File: package.json [hunk 3]
```diff
@@ -13,10 +13,8 @@
     ],
     "homepage": "https://cezerin.org",
     "bugs": "https://github.com/cezerin2/cezerin2-admin/issues",
-    "license": "MIT",
-    "author": {
-        "name": "Restmount <jsonmount@gmail.com>"
-    },
+    "license": "Unlicense",
```
Comment (ngoyal16) [inline package.json:16]: why changes in this

File: LICENSE (hunk 2 above)
Comment (Himadu2000) [reply LICENSE:1]: Sorry wrong commit for that. I will change them....

File: CONTRIBUTING.md (hunk 1 above)
Comment (Himadu2000) [reply CONTRIBUTING.md:1]: Sorry wrong commit for that. I will change them....

File: package.json (hunk 3 above)
Comment (Himadu2000) [reply package.json:16]: Sorry wrong commit for that. I will change them....

**Pull request merged**